apps/src/storage/dataBrowser/DataVisualizer.jsx

```diff
@@ -108,9 +103,25 @@ class DataVisualizer extends React.Component {
     }
   };
 
+  findNumericColumns = records => {
+    return this.props.tableColumns.filter(
+      column =>
+        records.filter(
+          record =>
+            typeof record[column] === 'undefined' ||
```

Comment:
Curious: Why do we include 'undefined' here?

Reply:
offline sync: We are filtering for "number columns." If a number column has all numbers with a couple blank cells, the column is still a number column. If, however, it somehow has a mix of numbers and strings, it is not a number column.